lib/attr_encrypted/adapters/active_record.rb

```diff
@@ -16,10 +16,9 @@ def reload_with_attr_encrypted(*args, &block)
             alias_method_chain :reload, :attr_encrypted
 
             attr_encrypted_options[:encode] = true
+
             class << self
-              alias_method :attr_encryptor, :attr_encrypted
```

Comment:
It's my understanding that this is already happening here: https://github.com/attr-encrypted/attr_encrypted/blob/master/lib/attr_encrypted.rb#L160

Have I misunderstood the something?